src/test/ui/consts/static_mut_containing_mut_ref2.stderr

```diff
@@ -1,8 +1,11 @@
-error[E0017]: references in statics may only refer to immutable values
+error[E0658]: references in statics may only refer to immutable values
   --> $DIR/static_mut_containing_mut_ref2.rs:3:46
    |
 LL | pub static mut STDERR_BUFFER: () = unsafe { *(&mut STDERR_BUFFER_SPACE) = 42; };
```

Comment:
turn on the feature gate in this test. It should still fail (or even better: make it a dual test for with and without the feature gate)